src/elements/element.line.js

```diff
@@ -2,10 +2,12 @@
 
 import defaults from '../core/core.defaults';
 import Element from '../core/core.element';
-import helpers from '../helpers';
+import {_bezierInterpolation, _pointInLine, _steppedInterpolation} from '../helpers/helpers.math';
```

Comment:
I wonder if these should be in a `helpers.interpolation`? They're a little more specific than what I'd expect to be in `helpers.math`

Reply:
done